docs/security.html

```diff
@@ -661,6 +661,108 @@ <h3><a id="security_sasl" href="#security_sasl">7.3 Authentication using SASL</a
             old mechanism from JAAS config file. Incrementally bounce the cluster again.</li>
         </ol>
     </li>
+
+    <li><h4><a id="security_delegation_token" href="#security_delegation_token">Authentication using Delegation Tokens</a></h4>
+        <p>Delegation token based authentication is a lightweight authentication mechanism to complement existing SASL/SSL
+            methods. Delegation tokens are shared secrets between kafka brokers and clients. Delegation tokens will help processing
+            frameworks to distribute the workload to available workers in a secure environment without the added cost of distributing
+            Kerberos TGT or keytabs. See <a href="https://cwiki.apache.org/confluence/display/KAFKA/KIP-48+Delegation+token+support+for+Kafka">KIP-48</a>
```

Comment:
This also saves the cost of distributing keystores when 2-way SSL is used.

Reply:
Thanks for the review. Updated the PR.